source/faucet/main.d

```diff
@@ -274,7 +274,8 @@ public class Faucet : FaucetAPI
         auto builder = TxBuilder(
             secret_keys.byKey().drop(uniform(0, secret_keys.length, rndGen)).front());
         builder.attach(utxo_rng);
-        return builder.sign(OutputType.Payment, 0, &this.keyUnlocker);
+        return builder.unlockSigner(&this.keyUnlocker)
+                      .sign(OutputType.Payment, 0);
```

Comment:
Extra whitespace.

Reply:
Where is the whitespace?

Reply:
Before the `.sign`. Should be like line 274.